src/dialects/mariadb/data-types.js

```diff
@@ -6,6 +6,7 @@ const momentTz = require('moment-timezone');
 const moment = require('moment');
 
 module.exports = BaseTypes => {
+  const warn = BaseTypes.ABSTRACT.warn.bind(undefined, 'https://msdn.microsoft.com/en-us/library/ms187752%28v=sql.110%29.aspx');
```

Comment:
This warn links to the documentation for mssql instead of mariadb

```suggestion
  const warn = BaseTypes.ABSTRACT.warn.bind(undefined, 'https://mariadb.com/kb/en/data-types/');
```